projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-jetpack-token-subscription-service.php

```diff
@@ -41,10 +55,26 @@ public function get_site_id() {
 	 * @return string The key.
 	 */
 	public function get_key() {
+		if ( $this->is_wpcom() ) {
+			// phpcs:ignore ImportDetection.Imports.RequireImports.Symbol
+			return defined( 'EARN_JWT_SIGNING_KEY' ) ? EARN_JWT_SIGNING_KEY : false;
+		}
 		$token = ( new Tokens() )->get_access_token();
 		if ( ! isset( $token->secret ) ) {
 			return false;
 		}
 		return $token->secret;
 	}
+
+	/**
+	 * Returns true if the current authenticated user is subscribed to the current site.
+	 *
+	 * @return boolean
+	 */
+	public function is_current_user_subscribed() {
+		if ( $this->is_wpcom() ) {
+			return false;
```

Comment:
Now that I am reading that, I am not sure about the `false` here 🤔

Reply:
in think a better approach would be to define this method in all the wpcom concrete classes so this never gets hit in wpcom.

Reply:
There's a `is_current_user_subscribed` function that overrides this one inside `WPCOM_Online_Subscription_Service`. I think my mistake here was adding this function at all. Instead of `return false`, it should just do what the parent function does. I'll remove this function altogether.